app/views/admin/loans/show.html.slim

```diff
@@ -36,3 +36,8 @@ section.loan.details class=(@loan.valid? ? 'show-view' : 'edit-view')
 
   div.loan-content
     = render "admin/loans/tabs/tabs"
+
+javascript:
+  $(function(){
+    new MS.Views.LoanTabView();
```

Comment:
You need to set `el` or Backbone won't work. So `new MS.Views.LoanTabView({ el: 'something' });`